cmd/openfga/openfga.go

```diff
@@ -56,6 +59,10 @@ type svcConfig struct {
 	// RequestTimeout is a limit on the time a request may take. If the value is 0, then there is no timeout.
 	RequestTimeout time.Duration `default:"0s" split_words:"true"`
 
+	EnableTLS   bool   `default:"false" split_words:"true"`
```

Comment:
I originally had `TLSEnable` and I didn't like it as it sounds weird. I am not sure I like `TLSEnabled` as it sounds like a statement rather than question. So in the end I went with `EnableTLS`. I am kind of torn.

Reply:
Yeah, I like `TLSEnabled` better. It is more of a statement, because it will always have a value (default false, otherwise true). FWIW I'm using `PlaygroundEnabled` for the embedded Playground 🤷 ..

Reply:
Well, if you both think so then I'll update it 🙂